packages/@aws-cdk/aws-batch-alpha/lib/ecs-container-definition.ts

```diff
@@ -546,6 +556,15 @@ export interface EcsContainerDefinitionProps {
    */
   readonly readonlyRootFilesystem?: boolean;
 
+  /**
+   * The operating system that your task definitions are running on.
+   *
+   * A runtimePlatform is supported only for tasks using the Fargate launch type.
+   *
+   * @default - Undefined.
+   */
+  readonly runtimePlatform?: RuntimePlatform;
```

Comment:
Instead of having this subproperty, how about we inline the 2 fields from `RuntimePlatform` here? It will be easier to work with for users.

[Prefer flat properties](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md#flat), CDK Design Guidelines, section *Props*.